Source/monster.cpp

```diff
@@ -1437,7 +1437,7 @@ void MonsterTalk(Monster &monster)
 			Quests[Q_VEIL]._qlog = true;
 		}
 		if (monster.talkMsg == TEXT_VEIL11 && (monster.flags & MFLAG_QUEST_COMPLETE) == 0) {
-			SpawnUnique(UITEM_STEELVEIL, monster.position.tile + Direction::South);
+			SpawnUnique(UITEM_STEELVEIL, monster.position.tile + Direction::South, true);

```

Comment:
sendmsg in SpawnUnique is true everywhere, maybe let's go with a default true value for it?

Reply:
Changed 🙂